src/main/java/gregtech/common/tools/ToolBuzzSaw.java

```diff
@@ -32,4 +33,9 @@ public ItemStack getBrokenStack(ItemStack stack) {
         IElectricItem electricItem = stack.getCapability(GregtechCapabilities.CAPABILITY_ELECTRIC_ITEM, null);
         return MetaItems.POWER_UNIT_LV.getChargedStackWithOverride(electricItem);
     }
+
+    @Override
+    public boolean canPlayBreakingSound(ItemStack stack, IBlockState state) {
+        return canMineBlock(state, stack);
```

Comment:
The buzzsaw should not play sounds when mining. It is indented for crafting and the chainsaw serves a better mining purpose instead. Additionally harvesting ice with it is extremely loud with sounds as it mines instantly.